build.sbt

```diff
@@ -7,55 +7,55 @@ import org.scalajs.sbtplugin.cross.CrossProject
 
 lazy val checkHeaders = taskKey[Unit]("Fail the build if createHeaders is not up-to-date")
 
-val monocleVersion = "1.2.1"
-val scalazVersion = "7.2.1"
-// Latest version built against scalacheck 1.12.5
 // doesn't support scala.js yet, until then tests are JVM-only
-val specs2Version = "3.7"
+val monocleVersion    = "1.3.0"
+val scalazVersion     = "7.2.6"
+val specs2Version     = "3.8.5"
+val scalacheckVersion = "1.13.2"
 
 val testDependencies = libraryDependencies ++= Seq(
-  "org.typelevel"  %% "discipline"                % "0.4"          % "test",
-  "com.github.julien-truffaut" %% "monocle-law"   % monocleVersion % "test",
-  "org.scalaz"     %% "scalaz-scalacheck-binding" % scalazVersion  % "test",
-  "org.typelevel"  %% "scalaz-specs2"             % "0.4.0"        % "test",
-  "org.specs2"     %% "specs2-core"               % specs2Version  % "test" force(),
-  "org.specs2"     %% "specs2-scalacheck"         % specs2Version  % "test" force(),
-  // `scalaz-scalack-binding` is built with `scalacheck` 1.12.5 so we are stuck
-  // with that version
-  "org.scalacheck" %% "scalacheck"                % "1.12.5"       % "test" force()
+  "com.github.julien-truffaut" %% "monocle-law"               % monocleVersion    % Test,
+  "org.scalaz"                 %% "scalaz-scalacheck-binding" % scalazVersion     % Test,
+  "org.typelevel"              %% "scalaz-specs2"             % "0.5.0-SNAPSHOT"  % Test,
+  "org.specs2"                 %% "specs2-core"               % specs2Version     % Test,
+  "org.specs2"                 %% "specs2-scalacheck"         % specs2Version     % Test,
+  "org.scalacheck"             %% "scalacheck"                % scalacheckVersion % Test
 )
 
-lazy val standardSettings = Seq(
+def universalSettings = Seq(
+  scalaVersion := "2.11.8",
+  organization := "com.slamdata",
+  licenses += ("Apache 2", url("http://www.apache.org/licenses/LICENSE-2.0")),
+  libraryDependencies ++= Seq(
+    "com.github.julien-truffaut" %%% "monocle-core" % monocleVersion,
+    "org.scalaz"                 %%% "scalaz-core"  % scalazVersion,
+    "com.github.mpilquist"       %%% "simulacrum"   % "0.9.0"
```

Comment:
Can you update this to 0.10.0? That lets us enable a couple more warts (and _hopefully_ also lets us use 0.10.0 in Quasar to do the same).